integration-tests/container/QA_TAG

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
What scheme do you envision for increasing the version ? My lack of opinion here would lead me use one plain integer for master, then something like `release-x.x-<integer>` for the release branches.